app/models/slide.rb

```diff
@@ -154,20 +156,28 @@ def validate_background_file
     end
 
     def validate_video_file(type, opts = nil)
-      video = File.open(self.send("#{type}").file.path)
-
-      if video.size > 12.megabytes
-        errors.add("#{type} Video: ", 'You cannot upload a video larger than 12 MB')
+      if !self.send("#{type}").file.nil?
```

Comment:
Ruby offers the `unless` conditional in place of `if !`. That is, I believe this could be:

    unless self.send("#{type}").file.nil?

Or perhaps more readably:

    if self.send("#{type}").file.present?